### PR TITLE
Rename `addons` --> `hub-addon`

### PR DESCRIPTION
**Pull request by dhaiducek**
Clarify subcommand referencing "addon" since the word is overloaded in OCM.

### CODE REVIEW

File: pkg/cmd/install/hubAddon/cmd.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // Copyright Contributors to the Open Cluster Management project
-package addons
+package hubAddon
```
Comment (JustinKuli) [inline pkg/cmd/install/hubAddon/cmd.go:2]: Usual go convention would be to keep this all lowercase: https://go.dev/blog/package-names

**Comment by dhaiducek** [comment]
Sorry for the noise--I didn't know Git was case insensitive for paths by default! Should be good now.

**Review by JustinKuli — APPROVED** [review]

**Pull request merged**